contextily/plotting.py

```diff
@@ -179,6 +180,7 @@ def add_attribution(ax, att=ATTRIBUTION, font_size=ATTRIBUTION_SIZE):
                           Matplotlib axis with `x_lim` and `y_lim` set in Web
                           Mercator (EPSG=3857) and attribution text added
     """
+    draw()
```

Comment:
can you add a comment here?

Reply:
A comment as in explaining where that comes from?

Reply:
Done

Reply:
Thanks!